README.md

```diff
@@ -11,16 +11,27 @@
 [![Rating](https://img.shields.io/visual-studio-marketplace/r/WatermelonTools.watermelon-tools?style=flat-square)](https://marketplace.visualstudio.com/items?itemName=WatermelonTools.watermelon-tools&ssr=false#review-details)
 [![Slack](https://img.shields.io/badge/Slack%20Community-Watermelon-brightgreen)](https://join.slack.com/t/watermelonusers/shared_invite/zt-15bjnr3rm-uoz8QMb1HMVB4Qywvq94~Q)
 
-Watermelon is an **open-source** integration between **GitHub** and **Visual Studio Code** to document code and view Git Blame. Watermelon makes you an expert on any file instantly by running `git blame` for you and telling you why a block of code was written that way by someone else.
+Watermelon is an **open-source** integration between **GitHub** and **Visual Studio Code** to help developers go beyond Git Blame. Watermelon makes you an expert on any file instantly by running `git blame` for you and telling you why a block of code was written that way by someone else.
 
 ## Features
 
-Watermelon has 2 actions that you can use:
+Watermelon has 3 main features that you can use:
 
 - View Git Blame
 - View Pull Requests
+- Daily Summary
 
-Simply click the Watermelon icon on the sidebar. After that you can click any of the 2 different buttons Watermelon has.
+The main UI to use Watermelon is the context box you see when you hover over lines of code with our extension installed. This UI is complemented with the sidebar we have.
+
+In the image below you can see how hovering over a line of code provides us the following data:
+
+![hover](https://i.ibb.co/rdvgVnS/hover-showcase.gif)
+
+- The most relevant PR for that line
```

Comment:
Is it there yet? 
That hasn't landed AFAIK

Reply:
Aren't we rendering a PR on the box?

Reply:
I looked at the image that was linked and I see the text "The latest commit is [...] by {user} on {date}", am I missing something?

Reply:
Ah, I see another hover, but it's only for the first 2 frames, hard to catch.
<img width="500" alt="Screen Shot 2022-07-13 at 6 32 43 PM" src="https://user-images.githubusercontent.com/11527621/178854340-d13c0602-0eeb-4892-b5d9-ebcb14e46a83.png">

Reply:
That is a Squash commit.

Reply:
If you run it, you will get a huge comment title with a lot of issue numbers (#352) for example.
That is a github specific behaviour

Reply:
Gotcha